lib/thermite/cargo.rb

```diff
@@ -114,6 +114,7 @@ def cargo_rustc_args
       args = []
       unless config.dynamic_linker_flags == '' || config.target_os == 'mingw32'
         args.push(
+          '--lib',
```

Comment:
cargo will only complain about this issue when handing additional flags to `rustc`. So it works just fine when no linker flags are passed.